src/test/java/redis/clients/jedis/tests/commands/JedisClusterCommandTest.java

```diff
@@ -376,48 +378,11 @@ protected void sleep(long sleepMillis) {
     try {
       testMe.run("");
       fail("cluster command did not fail");
-    } catch (JedisClusterMaxAttemptsException e) {
+    } catch (JedisClusterException e) {
```

Comment:
Test fails, changing the exception type fixes it:

```suggestion
    } catch (JedisClusterMaxAttemptsException e) {
```